server/routes/place/html.py

```diff
@@ -274,9 +318,52 @@ def place(place_dcid=None):
   # Block pages from being indexed if not on the main DC domain. This prevents
   # crawlers from indexing dev or custom DC versions of the place pages.
   block_indexing = not is_canonical_domain(flask.request.base_url)
-  logging.info(f"flask.requests.base_url is {flask.request.base_url}")
-  logging.info(f"Block indexing on place pages? {block_indexing}")
 
+  # Use SEO experiment templates on English overview pages for places in
+  # the experiment group.
+  use_experiment = place_dcid in SEO_EXPERIMENT_DCID_TO_HTML.keys(
```

Comment:
Nit: extract all of the logic to see if the experiment is enabled to a function. then you wont need two flags use_experiment/is_experiment_enabled_flask_env

Reply:
Good idea, extracted out this logic to a function.